projects/canopy/src/lib/brand-icon/brand-icons.interface.ts

```diff
@@ -4,161 +4,168 @@ export const lgBrandIconCalendar: {
+  data: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 320 320"><g id="Foundations/Icons/Brand-Icons/Calendar" fill="none" fill-rule="evenodd" stroke="none" stroke-width="1"><path id="lg-icon-fill-primary" fill="#FFD500" d="M24 44h272a4 4 0 0 1 4 4v60H20V48a4 4 0 0 1 4-4z"/><path id="Outlines" fill="#000" d="M260 16c2.208 0 4 1.863 4 4.159v19.84l31.261.001c4.74 0 8.599 3.832 8.735 8.613l.004.261v246.252c0 4.814-3.774 8.732-8.481 8.87l-.258.004H24.74c-4.738 0-8.599-3.832-8.735-8.613l-.004-.261V48.874c0-4.814 3.776-8.732 8.481-8.87L24.74 40 56 39.999v-19.84C56 17.863 57.792 16 60 16s4 1.863 4 4.159v19.84h32v-19.84C96 17.863 97.792 16 100 16s4 1.863 4 4.159v19.84h32v-19.84c0-2.296 1.792-4.159 4-4.159s4 1.863 4 4.159v19.84h32v-19.84c0-2.296 1.792-4.159 4-4.159s4 1.863 4 4.159v19.84h32v-19.84c0-2.296 1.792-4.159 4-4.159s4 1.863 4 4.159v19.84h32v-19.84c0-2.296 1.792-4.159 4-4.159zm36 96H24v183.675c0 .156.109.288.253.318l.064.007H295.68a.321.321 0 0 0 .313-.26l.006-.065V112zm-40 24c.754 0 1.46.222 2.062.608 1.164.707 1.938 1.962 1.938 3.392l-.002.11.002.14v127.497l-.002.142.002.111a4 4 0 0 1-4 4H64c-.76 0-1.472-.226-2.077-.618-1.156-.71-1.923-1.96-1.923-3.382l.002-.108-.001-.072L60 140.25l.001-.142A3.945 3.945 0 0 1 60 140c0-1.42.764-2.668 1.916-3.378A3.805 3.805 0 0 1 64 136h192zm-132 72H68v56h56v-56zm64 0h-56v56h56v-56zm64 0h-56v56h56v-56zm-12 40a4 4 0 1 1 0 8 4 4 0 0 1 0-8zm-32 0a4 4 0 1 1 0 8 4 4 0 0 1 0-8zm16-16a4 4 0 1 1 0 8 4 4 0 0 1 0-8zm16-16a4 4 0 1 1 0 8 4 4 0 0 1 0-8zm-32 0a4 4 0 1 1 0 8 4 4 0 0 1 0-8zm-84-72H68v56h56v-56zm64 0h-56v56h56v-56zm64 0h-56v56h56v-56zm43.68-96L264 47.999v19.842c0 2.296-1.792 4.159-4 4.159s-4-1.863-4-4.159V48h-32v19.842c0 2.296-1.792 4.159-4 4.159s-4-1.863-4-4.159V48h-32v19.842c0 2.296-1.792 4.159-4 4.159s-4-1.863-4-4.159V48h-32v19.842c0 2.296-1.792 4.159-4 4.159s-4-1.863-4-4.159V48h-32v19.842c0 2.296-1.792 4.159-4 4.159s-4-1.863-4-4.159V48H64v19.842C64 70.137 62.208 72 60 72s-4-1.863-4-4.159V48H24.317a.32.32 0 0 0-.31.26l-.007.065V104h272V48.325a.323.323 0 0 0-.32-.325z"/></g></svg>`,
```

Comment:
Not sure why it shows here that something is changed 🤔

Reply:
Try doing an `npm i`, we recently updated the packages which create and optimise these interfaces, so you may have a slightly older version.

Reply:
Done :) 
Thanks!!